Tweeter/Controllers/AccountController.cs

```diff
@@ -149,9 +150,10 @@ public ActionResult Register()
         [ValidateAntiForgeryToken]
         public async Task<ActionResult> Register(RegisterViewModel model)
         {
-            if (ModelState.IsValid)
+            TweeterRepository tr = new TweeterRepository();
```

Comment:
this repository instantiates every time we hit this method... let's move it up to be a class-level data member (such as `private TweeterRepository tr = new TweeterRepository();` at the opening of the class